defaults.js

```diff
@@ -2,9 +2,12 @@ module.exports = {
   modules: {
     '@apostrophecms/error': {},
     '@apostrophecms/util': {},
-    '@apostrophecms/http': {},
     '@apostrophecms/task': {},
+    '@apostrophecms/schema': {},
```

Comment:
Is the order here important? Can we rely on that?

Reply:
The order here is super important and yes, we can rely on it. This is the order in which modules are initialized and has evolved carefully over the years so they don't start up without their requirements already started up.

Reply:
You can think of `defaults.js` as the prologue to `app.js`, it's the implicit part.